gke/variables.tf

```diff
@@ -14,13 +14,13 @@ variable "creds-file" {
   default = "creds/imposing-union-227917.json"
 }
 
-variable "ssh_username" { }
-variable "ssh_password" { }
+variable "ssh_username" {}
+variable "ssh_password" {}
 
 output "gcp_cluster_endpoint" {
```

Comment:
I prefer to have outputs in a separate file vs putting them in `variables.tf`

Reply:
I did that for the module itself but was lazy here, it's a good thing to do :+1: